### DISCUSSION TITLE
Oferty pracy w firmach na azure

### DISCUSSION THREAD

**Discussion by gondar** [AzureCommunityPL/group#74] [Inspiracje]
Hej! Zastanawiam sie czy maja one swoje miejsce (lub podmiejsce) gdzies tutaj? No i czy nie powinnismy ustalic jakichs sensownych minimalnych regol wysylania takich propozycji?

**Comment by kaluzaaa** [comment]
Dobry pomysł. W sumie można zrobić jeden wątek i go przypiąć! :-)

**Reply by wi5nia** [reply]
Generalnie to czekamy na mozliwosc tworzenia wlasnych kategori... maja byc SOON i mysle, ze zrobimy osobna kategorie wtedy na oferty pracy i tez w [FAQ](https://github.com/AzureCommunityPL/group/tree/master/faq) beda wytyczne publikacji takich ofert.
